changing Ash challenge to reflect the assignments (#50)
4TUResearchData-Carpentries/interoperability-climate-sciences: episodes/introduction.md

:::::::::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::: challenge

### True/False or Agree/Disagree with discussion afterwards 

- “As long as data are open access, they are interoperable.”
- “Metadata standards help ensure interoperability.”

> - “As long as data is using an open standard format is interoperable” 

Reviewer: ```suggestion
- “As long as data are using an open standard format, they are interoperable.” 
```

Good to use either plural or singular with "data" across all the materials.  I'd go for plural

:::::::::solution

### Solution

F,T,F

:::::::::::::::::

:::::::::: instructor

> This exercise is for discussion in Plenum nad it can serves as a good link to the next section

Reviewer: ```suggestion
This exercise is for discussion in Plenum and it serves as a good link to the next section.
```

:::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::

> ## Why to bother to make datasets interoperable?

Reviewer: ```suggestion
## Why bother making datasets interoperable?
```

Or alternatively `##Why does dataset interoperability matter?` or `Why is it important to make datasets interoperable?`

- **Large data volume and velocity**: Earth observation missions (e.g., Sentinel, GOES), reanalysis products (ERA5), and high-resolution climate simulations produce terabytes to petabytes of data, making efficient, interoperable access necessary.

- **Different access mechanisms and services**: Data are distributed across portals using APIs, OPeNDAP servers, cloud object storage, FTP, THREDDS catalogs, proprietary download tools, or manual interfaces, requiring technical interoperability to automate workflows.

- **Versioning and reproducibility issues**: Climate datasets evolve frequently (e.g., reprocessed satellite series, new CMIP6 versions), and without stable identifiers or catalog metadata, reproducibility becomes difficult across institutions.

- **Need for multi-model and multi-dataset comparisons**: Studies such as model evaluation, bias correction, and data assimilation depend on aligning diverse datasets that were never originally designed to work together.

> ![*Image created with AI*](../episodes/fig/specific_challenges.png)

Reviewer: consider alt-text
